### PR TITLE
Travis/add mic connection

### PR DESCRIPTION
**Pull request by TravS2001**
Currently still trying to create a web connected microphone that will record on button press and return a .wav file. I have yet to find a way that works with the current packages that does not cause the system to crash. 

With the shared implementation I am getting an issue stating that NativeAppEventEmitter cant be resolved. This is something from a react-native-web package that is needed to use a expo-av mic package. 
In short, this dont work and the web doesnt have an answer.

I have tried various implementations, some causing this error, and others causing similar errors with different packages.

### CODE REVIEW

File: .env
Comment (Trev4388) [inline .env:1]: We should not add this file to the git repo for security reasons since it contains API keys. Instead, I think that each of us will just have to maintain a version locally on out own machines. 

Use the command `git rm .env --cached` to remove the file from git and keep it locally.

You may also want to see [this commit](https://github.com/pOrtiz01/gpt-interface/pull/4/commits/c4be1ced9d93b01b74d28b3948f3dd6a6703c55e) for how to add this to the `.gitignore` file so that git does not track it (then there is no risk of accidentally pushing it).
